packages/devtools/lib/src/inspector/inspector_tree.dart

```diff
@@ -376,9 +381,11 @@ abstract class InspectorTreeNode {
   }
 
   /// TODO(jacobr): move this method to the InspectorTree class.
-  InspectorTreeRow getRow(int index, {InspectorTreeNode selection}) {
+  // Use [getCachedRow] wherever possible, as [getRow] is slow and can cause
```

Comment:
Everything before the TODO looks like like a doc comment instead of a `//` comment.